js/core/devices.d.ts

```diff
@@ -142,7 +142,7 @@ declare class DevicesObject {
      * @prevFileNamespace DevExpress.core
      * @public
      */
-    orientation(): string;
+    orientation(): 'portrait' | 'landscape' | undefined;
```

Comment:
should I make the return value more specific or the `string` is enough?

Reply:
I think more specific is good